style.css

```diff
@@ -117,15 +119,268 @@ a {
   border-top: 34px;
   height: 20px;
   padding: 0;
+  margin-bottom: 62px;
 }
 
 .socials li {
   padding: 10px 10px 10px 10px;
 }
 
+.recent-works {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] The main container of this section should have a `display: grid;`  kindly, change the flex by the CSS grid. Let's rock, fix it 🧑‍🔧